.gitignore

```diff
@@ -12,6 +12,7 @@ static
 gladys-backups
 qemu-*
 coverage
+persist
```

Comment:
What is persist?

Reply:
`persist/` is a folder created by the lib `hap-node` to store bridge and accessories configuration, in prod (with docker) it stored in `/var/lib/gladysassistant/homekit` (define here https://github.com/GladysAssistant/Gladys/pull/1591/files#diff-e49a6d70a5acaa68174761271b7d850f6971bfaab7d0d150912f411acc585ebbR22). In dev it create persist/ folder in server root folder

Reply:
Good for me! Thanks :)